docs/guidance/model/milestones.md

```diff
@@ -0,0 +1,139 @@
+# Milestones
+
+Milestones can be included within the planning, tender, contract and contract implementation sections. 
+
+* The planning milestone block may describe anticipated milestones during the planning stage, such as the preparation 
+of key studies. These anticipated milestones usually set `milestone/type` to 'preProcurement'.
```

Comment:
Remove line-break, remove normative keyword, pluralize 'milestones', clarify that the planning milestones block serves two purposes, add examples for clarity:
```suggestion
* The planning milestones block is used for two types of milestones:
  * Key events in the planning stage, for example, the preparation of an environmental impact assessment, the approval to proceed with a project, or the date of a public consultation. 
 * Anticipated milestones during the contract implementation stage, for example, the date by which goods delivery of the goods is required.
```